graphite_api/functions.py

```diff
@@ -2711,14 +2711,18 @@ def smartSummarize(requestContext, seriesList, intervalString, func='sum'):
 
     # Adjust the start time to fit an entire day for intervals >= 1 day
     requestContext = requestContext.copy()
+    # ISSUE #51: Added tzinfo to requestContext in graphite_api/app.py:328
+    tzinfo = requestContext.get('tzinfo', None)
```

Comment:
Ah right -- the unit test should reproduce what the app does, so if tzinfo is provided to all functions then tests should do the same… Feel free to update tests when this parameter is required :)